src/mathjax/mathml-to-svg.ts

```diff
@@ -17,12 +17,34 @@
 import 'mathjax-full/js/util/entities/all'
 
 import { HTMLAdaptor } from 'mathjax-full/js/adaptors/HTMLAdaptor'
+import { MmlFactory } from 'mathjax-full/js/core/MmlTree/MmlFactory'
 import { HTMLDocument } from 'mathjax-full/js/handlers/html/HTMLDocument'
 import { MathML } from 'mathjax-full/js/input/mathml'
 import { SVG } from 'mathjax-full/js/output/svg'
+import { MmlNodeClass } from 'mathjax-full/ts/core/MmlTree/MmlNode'
+import { PropertyList } from 'mathjax-full/ts/core/Tree/Node'
 
 import { xmlSerializer } from '../transformer'
 
+class MyMmlFactory extends MmlFactory {
```

Comment:
Can we call this `ManusctiptsMmlFactory` or something like that?

Reply:
fixed